graphql-api/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/GraphQLMetricHandler.java

```diff
@@ -54,8 +64,21 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
       Tags tags = Tags
           .of("api", method.getDeclaringClass().getSimpleName())
           .and("method", method.getName())
-          .and("result", success ? "success" : "failure");
+          .and("result", success ? "success" : "failure")
+          .and("authentication_type", AuthenticationType.of(authenticationSupplier.get()).name());
       registry.timer("graphql_api", tags).record(stopwatch.elapsed());
     }
   }
+
+  enum AuthenticationType {
```

Comment:
I would strongly consider renaming this (and the tag). To me the 'authentication type' would be one of basic/Password/federated etc.

E.g, RoleType or similar.